spring-integration-core/src/main/java/org/springframework/integration/util/UUIDConverter.java

```diff
@@ -89,7 +94,7 @@ private static UUID fromStringBytes(String input) {
 		return UUID.nameUUIDFromBytes(input.getBytes(StandardCharsets.UTF_8));
 	}
 
-	private static byte[] serialize(Object object) {
+	private static byte @Nullable [] serialize(@Nullable Object object) {
```

Comment:
Don't we need to add `@Contract("null -> null; !null -> !null")` on this method as well?